source/common/http/conn_manager_impl.h

```diff
@@ -182,7 +221,7 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
       StopAllBuffer,       // Iteration has stopped for all frame types, and following data should
                            // be buffered.
       StopAllWatermark,    // Iteration has stopped for all frame types, and following data should
-                           // be buffered until high watermark is reached.
+      // be buffered until high watermark is reached.
```

Comment:
Not sure if the format stuff will catch this, but if not: please re-align

Reply:
done.